02_docker/multistage/task08/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  uploader:
+    build: ./uploader
+    ports:
+      - "5001:5001"
+    volumes:
+      - shared_files:/app/files
+  
+  downloader:
+    build: ./downloader
+    ports: 
+      - "5000:5000"
+    volumes:
+      - shared_files:/app/files
+
+volumes:
+  shared_files:
+    driver: local
+
```

Comment:
Finde es gut, dass du hier eine Compose verfasst hast, die Applikation gefällt mir auch sehr gut.
Ein paar Verbesserungsvorschläge, speziell für produktive bzw. development Compose Dateien wären noch:

- nutze ENV-Variablen, da wo es Sinn macht, in diesem Fall könnte man bspw. noch den FLASK-Server mit seiner Runtime-Umgebung versehen:
```
    environment:
      - FLASK_ENV=testing/produvtion/development
```

- eine restart policy erspart speziell den Entwicklern auch noch viel Troubleshooting:
`    restart: unless-stopped`

- ein Healthcheck, der als Quick & Dirty Smoketest agiert, zeigt sofort ob die Applikation funktionall ist oder nicht
```
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:5001"]
      interval: 30s
      timeout: 10s
      retries: 3
```